unstructured/metrics/evaluate.py

```diff
@@ -116,7 +116,13 @@ def measure_text_extraction_accuracy(
             except Exception:
                 # if any of the output/source file is unable to open, skip the loop
                 continue
-            accuracy = round(calculate_accuracy(output_cct, source_cct, weights), 3)
+            # NOTE(amadeusz): Levenshtein distance calculation takes too long
+            # skip it if file sizes differ wildly
+            if 0.5 < len(output_cct.encode()) / len(source_cct.encode()) < 2.0:
+                accuracy = round(calculate_accuracy(output_cct, source_cct, weights), 3)
+            else:
+                # 0.01 to distinguish it was set manually
+                accuracy = 0.01
             percent_missing = round(calculate_percent_missing_text(output_cct, source_cct), 3)
```

Comment:
Do we want to calculate `percent_missing` still, if we are skipping `accuracy`?

Reply:
We can but for those examples accuracy was always `0.0` but percent_missing wasn't `1.0` so it would change current results.
And it's quicker to calculate than accuracy.

Reply:
Maybe a question for @cragwolfe ?

Reply:
yeah, assuming it is much quicker to calculate than accuracy, i'm good to leave `percent_missing` as is

Reply:
Thank you. Then this looks good to me!